membership-attribute-service/app/controllers/AccountController.scala

```diff
@@ -40,12 +41,11 @@ object AccountHelpers {
   case object NoFilter extends OptionalSubscriptionsFilter
 
   def subscriptionSelector[P <: SubscriptionPlan.AnyPlan](
-      subscriptionNameOption: Option[memsub.Subscription.Name],
+      subscriptionName: memsub.Subscription.Name,
```

Comment:
this doesn't need to be optional any more as the "old" update contribution amount endpoint that doesn't specify which subscription is not called in the last fortnight